### DISCUSSION TITLE
Upcoming Q&A webinar on Logtalk features, libraries, and tools

### DISCUSSION THREAD

**Discussion by pmoura** [LogtalkDotOrg/logtalk3#132] [Announcements]
Time to plan the next Logtalk webinar, which will focus on Q&A about Logtalk features, libraries, and tools. As always, the most difficult bit is finding a suitable day and hour with interested users in different time zones. My initial proposal is next Monday, January 31, at 20:00 UTC+00:00, with an expected duration of one hour. Please tell us here if you plan to attend and if the time slot works for you. As before, the webinar is open to both sponsors and guests (let us know if you need an invite).

**Comment by hnbeck** [comment]
Thats great! Timeslot would be ok, one Week later (07.02) may be a little bit better for me. But 31.01. is ok, too.

I would attend. My working areas (and therefore questions) are related to 

- XML, 
- relation of libraries coming with Logtalk and libraries of SWI Prolog -> portability, 
- components and the things ^^ and ::  and memory (retract/assert) 
- networking and multithreading

I work on Win10, Visual Studio Code and Logtalk 3.52 and SWI Prolog 8.2.4

Cheers

Hans

**Comment by jacobfriedman** [comment]
The timeslot is fine for me! I'm in.

**Comment by aarroyoc** [comment]
I suppose I can join as a guest. Timeslot is fine for me too. I haven't done anything big in Logtalk yet I've played a little bit and I want to know more

**Reply by pmoura** [reply]
Consider yourself invited. 🙂

**Comment by mlliarm** [comment]
Works for me too :).

**Comment by PaulBrownMagic** [comment]
Works for me!

**Comment by pmoura** [comment]
Thanks for all the feedback. The webinar will be next Monday, January 31, at 20:00 UTC+00:00. Sponsors and guests will receive an email with the connection details. If you're a guest or want to attend as a guest, make sure I have your email address so that I can also share with you the connection details.
